packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/src/api/dataProducts/operations.ts

```diff
@@ -3,10 +3,10 @@
 
 import { NetworkAnalyticsApiContext as Client } from "../index.js";
 import {
+  errorResponseDeserializer,
```

Comment:
@qiaozha Could we fix the import order? It seems the order is unstable and we need to regen serveral times to match with sequence in ci, the most bad case is we can't produce same order with ci verison.